GitHubDashboard/Controllers/QueryController.cs

```diff
@@ -132,6 +132,39 @@ private async Task<IReadOnlyList<Issue>> GetIssuesAsync(string owner, string rep
             // Catch it in the calling function...
             var issues = await _gitHubClient.Issue.GetAllForRepository(owner, repository, issueRequest);
             issues = issues.Where(i => i.PullRequest == null).ToList();
+
+            // We now need to exclude the milestone
+            if (!String.IsNullOrEmpty(excludedMilestone) && !(excludedMilestone == "undefined"))
+            {
+                issues = issues.Where(i => i.Milestone == null || i.Milestone.Title != excludedMilestone).ToList();
+            }
+
+            // We now need to exclude all the issues that have labels that should be excluded
+            if (!String.IsNullOrEmpty(excludedLabels) && !(excludedLabels == "undefined"))
+            {
+                var filteredIssues = new List<Issue>();
+                var excludedLabelValues = excludedLabels.Split(',');               
+
+                foreach(Issue i in issues)
```

Comment:
I'm okay with `var` in `foreach` loops.

Reply:
Also, nit: Prefer space between `foreach` and `(` and between `if` and `(`.

Reply:
fixed

---
In reply to: [247579994](https://github.com/Pilchie/GitHubDashboard/pull/13#discussion_r247579994) [](ancestors = 247579994)

Reply:
I am not sure I understand this comment. Could you expand?

---
In reply to: [247579324](https://github.com/Pilchie/GitHubDashboard/pull/13#discussion_r247579324) [](ancestors = 247579324)

Reply:
You already fixed it.  You had a few places where you had `foreach(` and `if(` instead of `foreach (` and `if (`.